src/macros.rs

```diff
@@ -4,6 +4,13 @@ macro_rules! define_uuid_macro {
         #[cfg(feature = "macro-diagnostics")]
         #[macro_export]
         macro_rules! uuid {
+            ($uuid:ident) => {{
```

Comment:
```suggestion
            ($uuid:expr) => {{
```

We can use expr here too.